### PR TITLE
added PR checklist.

### PR DESCRIPTION
**Pull request by edeyoung**
JIRA ticket: https://oncprojectracking.healthit.gov/support/browse/LANTERN-3

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**Primary Reviewer:**
+
+- [ ] Code works.
+  - Code has been run locally and the basic functionality affected by the PR has been checked.
+  - Edge cases have been checked to ensure behavior is as expected.
+- [ ] Code review has been performed.
+  - Code is maintainable, reusable, efficient, and correct.
+  - Code accomplishes the tasks purpose.
+  - Code follows style guidance appropriate for the language.
```
Comment (bmath10) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should running the linter be explicitly stated somewhere or does it just fall under style guidance?

**Comment by bmath10** [comment]
I don't know if this should be specified here or somewhere else, but should the branch be deleted after it's merged in? or is there some set amount of time you want to keep that branch around?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+**Submitter:**
+- [ ] Administrative tasks are complete.
+  - This pull request describes why these changes were made.
+  - The JIRA ticket links to this PR.
+  - The JIRA ticket for this PR is: 
+- [ ] Tests are complete.
+  - Tests are included and test edge cases.
+  - Tests have been run locally and pass.
+- [ ] Code review has been performed.
+  - Code diff has been done and checked to ensure only expected code is being committed.
```
Comment (bmath10) [inline .github/PULL_REQUEST_TEMPLATE.md]: On a previous project I was on, it was a requirement to make sure your feature branch has rebased off of master to make sure you have all the latest changes before merging. That project in general favored rebasing and rewriting history over merging though; what's your opinion on rebasing vs merging?
